scripts/tests/cirque_tests.sh

```diff
@@ -49,6 +49,9 @@ CIRQUE_TESTS=(
     "CommissioningFailureOnReportTest"
     "PythonCommissioningTest"
     "CommissioningWindowTest"
+    "SubscriptionResumptionTest"
```

Comment:
Thse seem to fail .... are these supposed to be added?

Reply:
OK, I will create another PR to fix this issue